### ISSUE TITLE
Tab is Not Focussed on First Click

### ISSUE THREAD

**Issue by thejamespower** [reactjs/react-tabs#272]
When clicking on a tab, it is expected that focus would be placed on that tab to instantly enable keyboard navigation. Currently this does not occur until the second click, or by pressing tab.

It is also worth noting that WAI-ARIA recommends handling roving tabindex like this: https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex

**Comment by flurmbo** [comment]
Hi, does anyone mind if I work on this issue?

**Comment by joepvl** [comment]
I'd say go for it! It's probably a good idea to detail what the current behaviour is (since the issue is from almost a year ago — although I don't think the behaviour has changed in the mean time) and how you'd like to change it.

**Comment by Morta1** [comment]
hey @flurmbo, any update about this issue?
I still need to click twice on the tab header to be able to navigate with the keyboard.

**Comment by arur** [comment]
This is still relevant. I guess there are no updates on this one.

**Comment by bestuiexperience** [comment]
I am also wondering if there will be a fix for this. I think this was working properly in an earlier version of React tabs. It is considered an accessibility violation not to display a focus ring around the active tab.
